app/views/mailers/notify_mailer/tag_moderator_confirmation_email.html.erb

```diff
@@ -17,10 +17,10 @@
   </li>
 </ul>
 <p><strong>For details on how to access these features, visit our
-  <a href="https://dev.to/tag-moderation">tag moderation</a> page.</strong></p>
+  <a href="<%= app_url("/tag-moderation") %>">tag moderation</a> page.</strong></p>
```

Comment:
where does this page come from? can't find it locally but it exists in production and it's not in the routes file 🤷‍♂️

Reply:
@rhymes It was probably created via the admin pages.

Reply:
magic!